<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>320</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>371</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
   
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */

    
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_13">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="nomor_hp">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>900000</width>
         <height>19</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{border: 2px solid gray;
border-radius: 6px;
padding: 0 8px;
min-height: 15px ;
max-height:15px;
}

QLineEdit:read-only {
    background: lightblue;
}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_14">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>Nama Pelanggan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>Nomor HP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>Alamat</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_16">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {color:black;border:0px;font: 15px &quot;Open Sans&quot;;
padding: 0 0px;}
</string>
       </property>
       <property name="text">
        <string>:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="nama_pelanggan">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90000</width>
         <height>19</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{border: 2px solid gray;
border-radius: 6px;
padding: 0 8px;
min-height: 15px ;
max-height:15px;
}

QLineEdit:read-only {
    background: lightblue;
}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="alamat_pelanggan">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>19</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90000</width>
         <height>19</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{border: 2px solid gray;
border-radius: 6px;
padding: 0 8px;
min-height: 15px ;
max-height:15px;
}

QLineEdit:read-only {
    background: lightblue;
}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QPushButton" name="apply">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {color:white; background-color:rgb(168, 168, 168);border:0px;font: 13px &quot;Open Sans&quot;;
border-radius: 7px;

padding: 0 8px;}


QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}


QPushButton:hover {
  background-color:rgb(61, 61, 61)
}

</string>
       </property>
       <property name="text">
        <string>Tambah Data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="email">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>9000</width>
         <height>19</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{border: 2px solid gray;
border-radius: 6px;
padding: 0 8px;
min-height: 15px ;
max-height:15px;
}


QLineEdit:read-only {
    background: lightblue;
}</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>apply</tabstop>
  <tabstop>nomor_hp</tabstop>
  <tabstop>email</tabstop>
 </tabstops>
 <resources>
  <include location="../data/ui/logo.qrc"/>
 </resources>
 <connections/>
</ui>
